students/WesWang/lesson2/gridprinter.py

```diff
@@ -0,0 +1,29 @@
+#=========================================================================================
+#Name: Grid Printer
+#Author: Wesley Wang
+#Date: 4/12/2018
+#=========================================================================================
+
+"""
+Used by print_grid to draw horizontal line(floor)
+:param: Number row/col, Size of grid
+:return: Floor with specified row/col count and size
+"""
+def draw_floor(rowcol, size):
+  for x in range(rowcol):
+      print("+" + "-"*size, end='')
```

Comment:
Cool, I like your helper function! Another option would be to use string multiplication instead of using a loop here, but this works too! (Too many loops within loops can get computationally expensive especially as the grid size and number of rows increases)